### ISSUE TITLE
[Bug] configure in laravel

### ISSUE THREAD

**Issue by ankit-kumar02** [creativetimofficial/bootstrap-vue-argon-dashboard#3]
### Version
laravel 6.0
### Reproduction link
http://nolink.com
### Operating System
windows 10
### Device
Laptop
### Browser & Version
google chrome/latest version
### Steps to reproduce
I didn,t understand how to configure bootstrap vue argon theme in laravel 6.0
### What is expected?
plz give me solution to set up theme in laravel
### What is actually happening?
didn,t understand

---
### Additional comments
plz help me

**Comment by krognome** [comment]
Yeah I updated to Laravel 10 and PHP 8.2 but I know that is not relative but installing into Laravel is very difficult maybe bc you guys do not have on npm? I was able to install via npm as well but actually working it into my web pack was an impossible nightmare. AI like git hub co pilot does not have solution even meaning there's no much out there on this is there?? I want to buy pro but if I can't get this going then why would I? Also is upgrading a current fully developed system using bootstrap Vue with this theme a good idea or would it have to be recreated if I want to use this theme or pro even?
